yarn.lock

```diff
@@ -9265,10 +9265,10 @@ minimist-options@4.1.0:
     is-plain-obj "^1.1.0"
     kind-of "^6.0.3"
 
-minimist@^1.2.0, minimist@^1.2.5:
-  version "1.2.5"
-  resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.5.tgz#67d66014b66a6a8aaa0c083c5fd58df4e4e97602"
-  integrity sha512-FM9nNUYrRBAELZQT3xeZQ7fmMOBg6nWNmJKTcgsJeaLstP/UODVpGsr5OhXhhXg6f+qtJ8uiZ+PUxkDWcgIXLw==
+minimist@1.2.6, minimist@^1.2.0, minimist@^1.2.5:
```

Comment:
Based on the `yarn.lock` there is no need to use resolution to force `minimist` to upgrade. `minimist@^1.2.0`, `minimist@^1.2.5` match both `minimist@1.2.6`. You can just remove this entry from the lockfile and run `yarn` to upgrade.